src/runtime/GlobalObject.cpp

```diff
@@ -1116,6 +1128,14 @@ void GlobalObject::installOthers(ExecutionState& state)
 
     m_stringProxyObject = new StringObject(state);
     m_numberProxyObject = new NumberObject(state);
+
+    // 8.2.2 - 12
+    // AddRestrictedFunctionProperties(funcProto, realmRec).
+    auto fn = new FunctionObject(state, NativeFunctionInfo(state.context()->staticStrings().caller, builtinCallerandArgumentsGetterSetter, 0, nullptr, NativeFunctionInfo::Strict));
```

Comment:
`NativeFunctionInfo` is changed by #319 not to include constructor function pointer, so please rebase this code to the master branch.

Reply:
I fixed it